gcloud-java-logging/src/main/java/com/google/cloud/logging/spi/v2/testing/LocalLoggingImpl.java

```diff
@@ -78,22 +76,6 @@ public void listLogEntries(
   }
 
   @Override
-  public void readLogEntries(
```

Comment:
Right, this method has been removed from LoggingServiceV2Grpc.LoggingServiceV2 so it is no longer needed.